sklearn/mixture/_gaussian_mixture.py

```diff
@@ -514,6 +514,15 @@ class GaussianMixture(BaseMixture):
 
     Read more in the :ref:`User Guide <gmm>`.
 
+    For examples on different methods of initialization, refer to:
+    :ref:`sphx_glr_auto_examples_mixture_plot_gmm_init.py`
```

Comment:
"For a detailed example of how the `init_params` parameter impacts the model's convergence behaviour, see <example_link>."